.github/actions/setup/action.yaml

```diff
@@ -0,0 +1,25 @@
+name: Setup Rust Builder
```

Comment:
Not very sure about the file naming. Seems this is just "do cargo cache", thus the name may be "cargo_cache" or something like that?